code/game/dna/dna_modifier.dm

```diff
@@ -184,8 +184,6 @@
 		if(M.Victim == L)
 			usr << "[L.name] will not fit into the DNA Scanner because they have a slime latched onto their head."
 			return
-	if(L == user)
-		return
 	visible_message("[user] puts [L.name] into the DNA Scanner.")
```

Comment:
Personally, I'd have it display a different message if the player puts themselves into the machine than showing this one in all cases.

Something like "[user] climbs into the DNA Scanner." would seem more consistent with other self-drag things like climbing onto tables or into sleepers.